net.xm1math.Texmaker.yaml

```diff
@@ -15,6 +15,8 @@ finish-args:
   - --device=dri
   - --filesystem=xdg-documents
   - --env=PATH=/app/texlive/bin/x86_64-linux:/app/texlive/bin/aarch64-linux:/app/bin:/usr/bin  # required to find texlive binaries
+  - --env=LD_LIBRARY_PATH=/app/texlive/lib:/app/texlive/lib/perl5/5.38.0/aarch64_linux/CORE:/app/texlive/lib/perl5/5.38.0/x86_64-linux/CORE
+  - --env=PERL5LIB=/app/texlive/lib/perl5/5.38.0:/app/texlive/lib/perl5/site_perl/5.38.0
```

Comment:
The perl version here might change I assume? I wonder if it'd be better to do this as a wrapper script like textstudio that gets the perl version https://github.com/flathub/org.texstudio.TeXstudio/blob/master/texstudio.sh

Also, a ${LD_LIBRARY_PATH}, ${PERL5LIB} would be appropriate if anything tries to set to them, I think.

Reply:
> The perl version here might change I assume? I wonder if it'd be better to do this as a wrapper script like textstudio that gets the perl version https://github.com/flathub/org.texstudio.TeXstudio/blob/master/texstudio.sh

Yes, the perl version might change but it does not very often. (Perl 5.34 was released in May 2021, Perl 5.46 in May 2022 and Perl 5.38 in July 2023)  But I don't know if that's worth a wrapper script.

> Also, a ${LD_LIBRARY_PATH}, ${PERL5LIB} would be appropriate if anything tries to set to them, I think. The $PATH variable should also be appended like that instead of setting `/app/bin`

What do you mean by "anything tries to set them"? I adapted them to the texlive extension and if someone installs Perl into their home directory, I can't predict the path anyway.
And all important executables lie in the $PATH. So why change it?

Reply:
The perl version would be easy to miss, if it changes. I think it is worth adding a wrapper.

I meant appending to existing environment variables, $PATH is already set in the sandbox to /app/bin and /usr/bin so doing `/new/path:${PATH}` in the wrapper would be better. Same for the other two.